docker/build.sh

```diff
@@ -59,6 +59,7 @@ docker buildx bake --no-cache --load --progress=plain -f "$SCRIPT_DIR/autoware-u
     --set "*.args.ROS_DISTRO=$rosdistro" \
     --set "*.args.BASE_IMAGE=$base_image" \
     --set "*.args.SETUP_ARGS=$setup_args" \
+    --set "*.args.CUDA_BASE_IMAGE=$cuda_base_image" \
```

Comment:
The cuda_base_image is given in the matrix condition: https://github.com/autowarefoundation/autoware/blob/964ad927cd6695cd6645dbafda6f43224f274307/.github/workflows/docker-build-and-push-main.yaml#L27

So it should be removed if you use that name.

```suggestion
```

Or I think you can rename it to `prebuilt_base_image`.
In that case, it might be good to also rename the normal `base_image` to `devel_base_image`. (But it should be considered in another PR.)

```suggestion
    --set "*.args.PREBUILT_BASE_IMAGE=$prebuilt_base_image" \
```

Reply:
Thanks for your advice, i will rename it to prebuilt_base_image